### ISSUE TITLE
md-datepicker aria-label error

### ISSUE THREAD

**Issue by barmaley443** [angular/material#9340]
**Actual Behavior**:
console error
ARIA: Attribute " aria-label ", required for accessibility, is missing on node: 

**CodePen** (or steps to reproduce the issue): *
http://codepen.io/barmaley443/pen/WxWbRV

**Angular Versions**: *
 -  `Angular Version:` 1.5.5
 -  `Angular Material Version:` 1.1.0

**Additional Information**:
 -  `Browser Type: *` Chrome
 -  `Browser Version: *` 52.0.2743.116 (64-bit)
 -  `OS: *` MacOS
